sampling_fo2/parser/__init__.py

```diff
@@ -8,7 +8,9 @@
 
 def parse_input(input_file: str) -> WFOMCSProblem:
     if input_file.endswith('.mln'):
-        raise NotImplementedError
+        with open(input_file, 'r') as f:
+            input_content = f.read()
+        return mln_parse(input_content)
```

Comment:
I would like to convert MLN to WFOMCS here, so that we can deal with counting and sampling in the same time.

Reply:
We can return with ORIGIN_PRED_SET for sampling.